tests/snapshots/run_single_files__match_mult_linearization.hvm.snap

```diff
@@ -0,0 +1,5 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/run_single_files/match_mult_linearization.hvm
+---
+λa match a { 0: λb λc λd (+ (+ b c) d); +: λf λg λh (+ (+ (+ h e) f) g) }
```

Comment:
This test result is incorrect, `e` is a free variable

Reply:
It's not incorrect, `e` is the variable `a-1` that got renamed by the unique_names pass. The succ arm contains the label information, but its not printed as it would be invalid syntax.

Reply:
Could you fix the readback by renaming the variable to the correct name?